### PR TITLE
Docs: Add Blockbridge volume plugin

### PR DESCRIPTION
**Pull request by srust**
Thank you for all your hard work on the volume plugins, and the volume plugins API!

We have just made available our volume plugin for Docker. This PR is to list the plugin alongside the other existing volume plugins in the plugins documentation.

Please consider this docs change for 1.8.

Thank you!

### CODE REVIEW

**Comment by thaJeztah** [comment]
Thanks! LGTM. I'll add it to the milestone, but @calavera has the final word in this :-)

/cc @moxiegirl @calavera (for review and cherry-pick)

File: docs/extend/plugins.md [hunk 1]
```diff
@@ -31,6 +31,12 @@ Follow the instructions in the plugin's documentation.
 
 The following plugins exist:
 
+* The [Blockbridge plugin](https://github.com/blockbridge/blockbridge-docker-volume)
```
Comment (moxiegirl) [inline docs/extend/plugins.md]: @srust Thank you for the contribution. Two sentences are a bit easier to read. 

--> https://gist.github.com/moxiegirl/48475ed3a43fac7aea4e

The [Blockbridge plugin](https://github.com/blockbridge/blockbridge-docker-volume)
  is a volume plugin that provides access to an extensible set of
  container-based persistent storage options. It supports single and multi-host Docker
  environments with features that include tenant isolation, automated
  provisioning, encryption, secure deletion, snapshots and QoS.

**Comment by srust** [comment]
@moxiegirl thank you for the review! You've got it. Two sentences, pushed a new commit.

**Comment by moxiegirl** [comment]
Thanks @srust LGTM! To @calavera for the final review and merge.

**Comment by calavera** [comment]
LGTM. Thank you!

**Comment by moxiegirl** [comment]
@calavera Cherry pick too?

**Pull request merged**